test/elixir_console_web/live/console_live_test.exs

```diff
@@ -20,7 +20,7 @@ defmodule ElixirConsoleWeb.ConsoleLiveTest do
     end
 
     test "command result is displayed", %{html: html} do
-      assert html =~ ~r/<div id="output.".*3.*<\/div>/s
+      assert html =~ ~r/<div.*id="output.".*3.*<\/div>/s
```

Comment:
Just a detail but by making this change now we're not necessarily matching the id="output" inside the div tag anymore.
I tested it changing in the history_component the output div by a different tag (section) and it still passes.
To scope it to the same div tag we need to change `.*` by `[^>]*`.

Reply:
Good call!